src/main/java/software/amazon/lambda/snapstart/ByteCodeIntrospector.java

```diff
@@ -43,7 +43,29 @@ private static Set<String> setOf(String ... strings) {
     };
 
     boolean isLambdaHandler(XClass xClass) {
-        return implementsLambdaInterface(xClass) || hasLambdaHandlerMethod(xClass);
+        return implementsLambdaInterface(xClass) || 
+               hasLambdaHandlerMethod(xClass) || 
+               (hasHandlerInClassName(xClass) && hasMethodCalledHandleRequest(xClass));
+    }
+
+    /**
+     * Returns true if the class has the word "handler" in the name.
```

Comment:
Another minor thing, but in the comment specify `"Handler"` with capital `H`

Reply:
Updated. Thanks